docs/docs/guide/09-interchange/03-walkthrough.md

```diff
@@ -5,382 +5,600 @@ description: Walkthrough of commands to use the interchain exchange module.
 
 # Use the Interchain Exchange
 
-In this chapter, you learn details about the order book and commands to:
+In this chapter, you will learn about the exchange and how it will function once
+it is implemented. This will give you a better understanding of what you will be
+building in the coming chapters.
 
-- Create an exchange order book for a token pair between two chains
-- Send sell orders on source chain
-- Send buy orders on target chain
-- Cancel sell or buy orders
+To achieve this, we will perform the following tasks:
 
-The next chapter contains the code for the implementation.
+* Start two local blockchains
+* Set up an IBC relayer between the two chains
+* Create an exchange order book for a token pair on the two chains
+* Submit sell orders on the Mars chain
+* Submit buy orders on the Venus chain
+* Cancel sell or buy orders
+
+Starting the two local blockchains and setting up the IBC relayer will allow us
+to create an exchange order book between the two chains. This order book will
+allow us to submit sell and buy orders, as well as cancel any orders that we no
+longer want to maintain.
+
+It is important to note that the commands in this chapter will only work
+properly if you have completed all the following chapters in this tutorial. By
+the end of this chapter, you should have a good understanding of how the
+exchange will operate.
+
+## Start blockchain nodes
+
+To start using the interchain exchange, you will need to start two separate
+blockchains. This can be done by running the `ignite chain serve` command,
+followed by the `-c` flag and the path to the configuration file for each
+blockchain. For example, to start the `mars` blockchain, you would run:
+
+```
+ignite chain serve -c mars.yml
+```
+
+To start the `venus` blockchain, you would run a similar command, but with the
+path to the `venus.yml` configuration file:
+
+```
+ignite chain serve -c venus.yml
+```
+
+Once both blockchains are running, you can proceed with configuring the relayer
+to enable interchain exchange between the two chains.
+
+## Relayer
+
+To set up an IBC relayer between two chains, you can use the `ignite relayer
+configure` command. This command allows you to specify the source and target
+chains, along with their respective RPC endpoints, faucet URLs, port numbers,
+versions, gas prices, and gas limits.
+
+```
+ignite relayer configure -a --source-rpc "http://0.0.0.0:26657" --source-faucet "http://0.0.0.0:4500" --source-port "dex" --source-version "dex-1" --source-gasprice "0.0000025stake" --source-prefix "cosmos" --source-gaslimit 300000 --target-rpc "http://0.0.0.0:26659" --target-faucet "http://0.0.0.0:4501" --target-port "dex" --target-version "dex-1" --target-gasprice "0.0000025stake" --target-prefix "cosmos" --target-gaslimit 300000
```

Comment:
The command asks me for a Source and Target account, and if I just use the default values, it ends up with an error: 

```
$ ignite relayer configure -a --source-rpc "http://0.0.0.0:26657" --source-faucet "http://0.0.0.0:4500" --source-port "dex" --source-version "dex-1" --source-gasprice "0.0000025stake" --source-prefix "cosmos" --source-gaslimit 300000 --target-rpc "http://0.0.0.0:26659" --target-faucet "http://0.0.0.0:4501" --target-port "dex" --target-version "dex-1" --target-gasprice "0.0000025stake" --target-prefix "cosmos" --target-gaslimit 300000
------
Setting up chains
------

? Source Account default
? Target Account default

🔐  Account on "source" is default(cosmos13gdkcvyz8us4u5lwdh7n5adldydrzmmgt0rfsw)

 |· received coins from a faucet
 |· (balance: 200000stake,10token)

✘ Rpc endpoint already exists with a different chain id
```

Reply:
Added a note about running `rm -rf ~/.ignite/relayer`.

https://github.com/ignite/cli/pull/3283/commits/290bbc65ed292a22b7e56fb14902e63f4a63d6fc

Reply:
\0/